src/components/PlantsPrimaryPage/index.tsx

```diff
@@ -85,10 +87,10 @@ export default function PlantsPrimaryPage({
   const setActivePlantingSite = useCallback(
     (site: PlantingSite | undefined) => {
       if (site) {
-        history.push(pagePath.replace(':plantingSiteId', site.id.toString()));
+        history.push({ pathname: pagePath.replace(':plantingSiteId', site.id.toString()), search: query });
```

Comment:
cool, is this to maintain the tabs?

Reply:
Yes! If not, when changing the selectedSite, the selected tab was lost